concept/Technical Concept.md

```diff
@@ -0,0 +1,18 @@
+Technical Concept
+=================
+
+These decisions have been made by the team:
+
+- Create a 2D game, because nobody of the team is familiar enough with 3D assets.
+
+- Use C#, because we are all very familiar with it.
+
+- Use Unity, as it popular, has a large ecosystem, and supports C# scripting as first class citizen.
+
+- Use a fixed aspect ratio for the game, because UI design and layouting would explode in effort otherwise.
+
+- Use 16:9 aspect ratio, because that's the most common.
+
+- Use mouse and keyboard for input devices (Reason?)
```

Comment:
Maybe because it's the most common for us. That does imply, that we target the PC platform primarily. We have NOT explicitly made the decision to target the PC platform though.

Reply:
Maybe also it take longer to find out how to implement other controls and test them so for start its easier to just take mouse and keyboard

Reply:
According to the Unity docs:
- Keyboard mouse
- Joysticks
- Controllers
- Touch screens
- And more...
Are all available through the Input Manager which is part of the core Unity platform and available by default.
https://docs.unity3d.com/Manual/Input.html

Reply:
Just because they are available doesn't mean they are easy to implement